### PR TITLE
Only collect metadata once per run instead of for each pair

### PR DESCRIPTION
**Pull request by dry923**
### Description

We are currently running metadata collection for each pair which is unneeded (especially they all under the same UUID). This PR sets METADATA_COLLECTION to false for each run after the first pair.

### CODE REVIEW

File: workloads/network-perf/run.sh [hunk 1]
```diff
@@ -36,11 +36,16 @@ if [[ ${SERVICEIP} == "true" && ${SERVICETYPE} == "metallb" ]]; then
...
 for pairs in ${PAIRS}; do
   export PAIRS=${pairs}
+  if [ $count -gt 0 ]; then
+    export METADATA_COLLECTION=false
+  fi
   if ! run_workload ${CR}; then
     exit 1
   fi
+  ((count++))
 done
```
Comment (rsevilla87) [inline workloads/network-perf/run.sh:49]: I think this way is easier (It doesn't need counter):

```suggestion
for pairs in ${PAIRS}; do
  export PAIRS=${pairs}
  if ! run_workload ${CR}; then
    exit 1
  fi
  export METADATA_COLLECTION=false
done
```

**Review by rsevilla87 — CHANGES_REQUESTED** [review]
Good idea, some comments above

File: workloads/network-perf/run.sh (hunk 1 above)
Comment (dry923) [reply workloads/network-perf/run.sh:49]: Good call. That's what I get for putting in a PR before coffee :-)

**Comment by rsevilla87** [comment]
Seems it needs some rebase @dry923

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**